src/js/components/Ballot/BallotScrollingContainer.jsx

```diff
@@ -141,6 +142,26 @@ class BallotScrollingContainer extends Component {
           <CandidateContainer
             data-modal-trigger
             className="u-cursor--pointer"
+            onClick={() => {
+              // DataLayer push
+              const dataLayerObject = {
+                event: 'relatedCandidateCardClick',
+                element_id: oneCandidate.we_vote_id,
+                candidateDetails: {
```

Comment:
I made some updates adjusting the information we want to pass about the Candidate in the candidateDetails dict in the Property Name spreadsheet https://docs.google.com/spreadsheets/d/1KO4_AOmddKvIpA79XrZdaVBdRkcP1HitqrvQkCo2A34/edit?pli=1&gid=1828322090#gid=1828322090